CPT331.Data.Migration/App.config

```diff
@@ -14,7 +14,7 @@
   <applicationSettings>
     <CPT331.Data.Migration.ApplicationConfig>
       <setting name="CrimeDataFolder" serializeAs="String">
-        <value>D:\Downloads\Crime Data</value>
+        <value>D:\Dropbox\COSC2650 Programming Project 1\Crime Data\XML Data Sources</value>
```

Comment:
Should be okay. 

I was thinking about using transformations (using https://visualstudiogallery.msdn.microsoft.com/579d3a78-3bdd-497c-bc21-aa6e6abbc859), but this doesn't overcome the need to check-in the transform file.

Not sure if there is a way to have user level transformations that aren't tied to the csproj file.

I have d:\, so I've just moved my dropbox folders 👍